src/contracts/markets/SiloMarket.sol

```diff
@@ -0,0 +1,192 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
The reasons we have this separate wrapper contract is to avoid needing to put `collectRewards` and `setHarvester` onto into the OriginArm contract? Does this have any other differences than just using the 4626 directly?

Reply:
That's correct. We only have `SiloMarket` so the Harvester can collect rewards. The 4626 functions just pass the calls through to the Silo vault which is also 4626.